Document/0x05b-Basic-Security_Testing.md

```diff
@@ -536,19 +572,113 @@ The output would look like this:
 [*] Finished heap search
 ```
 
-You can also use Java's reflection capabilities. To list the public methods of the `android.view.View` class, you could create a wrapper for this class in Frida and call `getMethods()` from the wrapper's `class` property:
+You can also use Java's reflection capabilities. To list the public methods of the `android.view.View` class, you could create a wrapper for this class in Frida and call `getMethods` from the wrapper's `class` property:
+
+```java
+Java.perform(function () {
+    var view = Java.use("android.view.View");
+    var methods = view.class.getMethods();
+    for(var i = 0; i < methods.length; i++) {
+        console.log(methods[i].toString());
+    }
+});
+```
+
+This will print a very long list of methods to the terminal:
 
 ```java
+public boolean android.view.View.canResolveLayoutDirection()
+public boolean android.view.View.canResolveTextAlignment()
+public boolean android.view.View.canResolveTextDirection()
+public boolean android.view.View.canScrollHorizontally(int)
+public boolean android.view.View.canScrollVertically(int)
+public final void android.view.View.cancelDragAndDrop()
+public void android.view.View.cancelLongPress()
+public final void android.view.View.cancelPendingInputEvents()
+...
+```
+
+###### Frida Bindings
+
+Frida can be also used from other languages which is very interesting for scripting purposes, the provided bindings include Python, C, NodeJS, and Swift.
```

Comment:
This sentence reads rather weird, i am not sure if my suggestion helps, but feel free to find something else otherwise:
```suggestion
Frida can be used with other languages for scripting purposes, such as Python, C, NodeJS, and Swift.
```

Reply:
New suggestion:

In order to extend the scripting experience, Frida offers bindings to programming languages such as Python, C, NodeJS, and Swift.